### ISSUE TITLE
Gutenboarding: improve designs picker accessibility

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#43555]
> At the Designs step VoiceOver will read the name of the option, and “Premium” tab if present, but nothing else describing the image, no alt text on the images. This can be difficult for users who rely on this tool or any other screen reader to actually be able to make a choice here, since all they would be going by are the random names we give te designs. Our recommendation would be to add some alt text to the images that explains the selling point of each design, like “Modern blog with soft colors and a call-to-action to subscribe” for Bowen for example.

![image](https://user-images.githubusercontent.com/87168/85380927-29180680-b546-11ea-8921-eadb514cadbb.png)

@ramonjd FYI, this can be added once we have descriptions for each design.

**Comment by ramonjd** [comment]
> this can be added once we have descriptions for each design.

We can add a placeholder in the [JSON file](https://github.com/Automattic/wp-calypso/blob/master/client/landing/gutenboarding/available-designs-config.json) for now.

Maybe fill it with some random copy, such as:

> Doyle intertwines classic post-modern overtones and titillating edges to assemble a fervent yet professional identity.

> Cassel emerges from a desire to impress, and is brimming with zest. Whispers of pastels and elegant serif fonts imbues atomic design elements with ardor and charm. 

I have more. :trollface:

**Comment by apeatling** [comment]
@simison I don't think these projects should follow a person, and stay with the team working on it.

**Comment by simison** [comment]
@apeatling oh sure, I had no intention for it to be carried over with a person. 😅  If I remember right, my reasoning was along the lines that this issue is actually about content and not about UI; so purely about our patterns/designs, just like the patterns have descriptions carried with them from the API, designs would need those too:

<img width="243" alt="image" src="https://user-images.githubusercontent.com/87168/110632684-97722700-81b0-11eb-9c52-35bd0957bc16.png">

The outcome is in the UI though so happy to have this on our backlog 👍  :-)
